### PR TITLE
Crazy typo check - THIS IS NOT WORKING!

### PR DESCRIPTION
**Pull request by harieshsathya**

### CODE REVIEW

File: src/main/java/com/pai/exchange/ExchangeClientFactory.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * Exchange specific client creation logic to generate clients.
+ */
+public class ExchangeClientFactory implements ClientFactory<ExchangeService> {
+
+    @Override
+    public ExchangeService getClient() {
+        ExchangeService service = new ExchangeService();
+        ExchangeCredentials credentials = new WebCredentials(
+                "olympus\\_peopleaisvc", "XL4LPQtj^Ktc8y", "owa.f5.com");
```
Comment (vood) [inline src/main/java/com/pai/exchange/ExchangeClientFactory.java]: no creds!!! please!!!

Comment (harieshsathya) [reply src/main/java/com/pai/exchange/ExchangeClientFactory.java]: Yes. I'm deleting this branch.

**Comment by vood** [comment]
Please also clean up the git history https://help.github.com/articles/removing-sensitive-data-from-a-repository/

**Comment by harieshsathya** [comment]
Let me delete the entire repository. Nothing committed yet and probably easier.

**Pull request closed without merging**